azext_iot/common/sas_token_auth.py

```diff
@@ -4,6 +4,11 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+"""
+sas_token_auth: Module containing Shared Access Signiture token class.
```

Comment:
Small typo 'Signiture'.

Reply:
fixed in commit: https://github.com/Azure/azure-iot-cli-extension/pull/4/commits/10de7bb46355c2c7be79c877a6d80a2eee20d536